### DISCUSSION TITLE
Lenny has violated GitHub Terms of Conduct

### DISCUSSION THREAD

**Discussion by PhilippeDunbar** [PrismLauncher/PrismLauncher#98] [General]
Github doesn't seem to have a way to directly report it, but I opened this in community discussion, let's blow it up so it gets attention from GitHub staff. Thanks.

https://github.com/community/community/discussions/36602

- A Leftest Queer Radicle

**Comment by IThundxr** [comment]
We know what has happened and github is doing what they can I recommend u don't start stuff like this pls thx
